.gitignore

```diff
@@ -13,4 +13,6 @@
 *.dllcache
 *.nipkg
 *.jslicache
-/node_modules
\ No newline at end of file
+/node_modules
+*.lvprojectcache
```

Comment:
We need to keep the .lvprojectcache files, please take it out of the .gitignore and check yours in